src/service_provider/service_provider.services.spec.js

```diff
@@ -0,0 +1,32 @@
+import { FindAllServiceProviders, FindAllServiceProvidersById, UpdateServiceProvider } from './service_provider.services'
+
+const fakeProvider = {
+    save: jest.fn(),
+    update: jest.fn(),
+    destroy: jest.fn()
+}
+
+const mockDb = {
+    service_provider: {
+        findAll: jest.fn(() => fakeProvider),
+        findOne: jest.fn(() => fakeProvider),
+        update: jest.fn(res => Promise.resolve([1, [{ id: 1, name: 'test' }]]))
+    }
+}
+
+describe('Testing Service Provider Services', () => {
```

Comment:
These tests don't seem like they're doing anything other than calling the mock and testing if you called the mock, which is always true

Reply:
Updated a few of them to make sure the functions are being called with the expected params.